.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
+* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it
+      should be force landed and forced-landing tag is set.
```

Comment:
"PR is green or force-landing label is set".  Again, we could check on this using the above trigger.